Formula/austin.rb

```diff
@@ -31,6 +31,6 @@ def install
 
   test do
     python3 = "python3.11"
-    shell_output(bin/"austin #{python3} -c \"from time import sleep; sleep(1)\"", 37)
+    shell_output(bin/"austin #{python3} -c \"from time import sleep; sleep(1)\"", 32)
```

Comment:
```suggestion
    shell_output(bin/"austin --usage", 0)
```

Reply:
yeah, I know it would work, but it is more dummy than the smoke test. The major question in here is why it fails with 3.5.0 build.

Reply:
> The major question in here is why it fails with 3.5.0 build.

On macOS, Austin 3.5 reports a "not Python process" error because of the new Python detection logic. In some cases, Austin would find the system's `bash` executable and try to call `task_for_pid` on it, which would fail under any circumstances (unless SIP is disabled). If we treated this as a fatal error it would prevent Austin from actually waiting to see if that turns into an actual Python process. So on macOS the error now turned from a permission issue to a "not Python" (that is, from 37 to 32). On Linux we don't have this problem. In theory, Austin should just work fine when spawning a command without sudo (sudo is only required when attaching to a process). It actually surprises me that one gets a permission error on Linux too, as this does not happen in Austin's CI runs, which also use GitHub action runners.

Reply:
the test was the same for macos/linux builds earlier, but now test runs successfully for linux. Let me update test to get this shipped.